package.json

```diff
@@ -43,17 +44,17 @@
   },
   "devDependencies": {
     "@biomejs/biome": "^2.2.3",
-    "@types/mocha": "^10.0.1",
     "@types/node": "^22.0.0",
-    "@types/sinon": "^17.0.0",
+    "@vitest/coverage-v8": "^3.2.4",
     "axios": "^1.2.1",
     "c8": "^10.0.0",
     "esbuild": "^0.25.9",
     "js-green-licenses": "^4.0.0",
-    "mocha": "^11.0.0",
     "nock": "^14.0.10",
-    "sinon": "^21.0.0",
-    "typescript": "~5.9.0"
+    "p-defer": "^4.0.1",
+    "typescript": "~5.9.3",
+    "unplugin-swc": "^1.5.8",
```

Comment:
whats up with unplugin-swc?

Reply:
Not necessary. We need to build anyway for the cjs test.